CMakeLists.txt

```
@@ -1,16 +1,18 @@
 cmake_minimum_required(VERSION 3.19)
 project(my_application LANGUAGES CXX)
 
-# Note: Use find_package(SixtyFPS) instead of the following three commands, if you prefer the package
-# approach.
-include(FetchContent)
-FetchContent_Declare(
+find_package(SixtyFPS)
+if (NOT SixtyFPS_FOUND)
+  # If SixtyFPS packages is not found, download from git
+  include(FetchContent)
```

I think it would be worthwhile to show a clarifying message, because by default when building not against a package, this will still produce messages that look like something is wrong.

```suggestion
find_package(SixtyFPS QUIET)
if (NOT SixtyFPS_FOUND)
  message("SixtyFPS could not be located in the CMake module search path. Downloading it from Git and building it locally");
  include(FetchContent)
```

Maybe it should be "QUIET" though but still continue to show the "error"?